src/macros.jl

```diff
@@ -614,19 +617,53 @@ function parseSDconstraint(_error::Function, args...)
 end
 
 """
-    @SDconstraint(m::Model, expr)
+    @SDconstraint(model::Model, expr)
 
 Add a semidefinite constraint described by the expression `expr`.
 
-    @SDconstraint(m::Model, ref[i=..., j=..., ...], expr)
+    @SDconstraint(model::Model, ref[i=..., j=..., ...], expr)
 
 Add a group of semidefinite constraints described by the expression `expr`
 parametrized by `i`, `j`, ...
 
 The expression `expr` needs to be of the form `a sign b` where `sign` is `⪰`,
 `≥`, `>=`, `⪯`, `≤` or `<=` and `a` and `b` are `square` matrices. It
-constrains that `a - b` (or `b - a` if the sign is `⪯`, `≤` or `<=`) is
-positive semidefinite.
+constrains the matrix `x = a - b` (or `x = b - a` if the sign is `⪯`, `≤` or
+`<=`) to be symmetric and positive semidefinite.
+
+If `x` is already symmetric, use `@constraint(model, Symmetric(x) in PSDCone())`
+instead to remove the need to compare the corresponding off-diagonal entries
```

Comment:
"remove the need" is a bit confusing (why is it needed?). We can say that by default, we check numerical symmetry of the matrix `x`, and if symmetry is violated by some arbitrary amount, we add explicit equality constraints. You can use `Symmetric(x)` to skip these checks if you know the matrix must be symmetric.